README.rst

```diff
@@ -202,4 +202,52 @@ again with the command::
 
     bors retry
 
-For more details, see `the bors reference docs <https://bors.tech/documentation/>`_
\ No newline at end of file
+For more details, see `the bors reference docs <https://bors.tech/documentation/>`_
+
+Releasing
+---------
+
+We follow the usual ``zest.releaser`` approach for releases.
+
+First:
+
+- Make sure the CHANGELOG is up-to-date and includes details about all the components included in the release
+
+.. code-block:: bash
+
+  $ git checkout -b prepare-X.Y.Z
+  $ prerelease
+
+Then:
+
+- Open a PR
+- When the PR is approved, merge it using bors
+
+Then:
+
+.. code-block:: bash
+
+   $ git checkout master
+   $ git pull
+   $ release
+   $ postrelease
+
+Then:
+
+The "Back to development" commit cannot be pushed to master because we don't allow pushes to master.
```

Comment:
There might be some zest.releaser options possible to avoid that (in setup.cfg IDK)

Reply:
There might be, but nothing easy. Anyhow, we could easily tell developers to `git reset --hard HEAD^` to get rid of it. But the absence of a changelog stub kind of bothers me.